### DISCUSSION TITLE
what is the guide tour component used in nationalmap and digitaltwin ?

### DISCUSSION THREAD

**Discussion by malnajdi** [TerriaJS/terriajs#5397] [Q&A] [answered]
Greeting Team and Community, 

I was working what is used to generate this component in the image below.
![image](https://user-images.githubusercontent.com/15247545/113952464-5e17f000-981e-11eb-9e58-400a88c78468.png)

**Comment by tephenavies** [comment, marked as answer]
Hi @malnajdi. This is part of TerriaJS version 8. See the `next` branch of [TerriaMap](https://github.com/TerriaJS/TerriaMap/tree/next) and [terriajs](https://github.com/TerriaJS/terriajs/tree/next).

The component is called [`TourExplanationBox`](https://github.com/TerriaJS/terriajs/blob/next/lib/ReactViews/Tour/TourPortal.jsx). Another relevant part of the feature are the `TourPoint`s (see [Sourcegraph search](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/TerriaJS/terriajs%24%40next+TourPoint&patternType=literal) for a start in seeing how it works).

**Reply by malnajdi** [reply]
@steve9164 Thanks a lot.
